app/src/main/java/com/xmartlabs/gong/ui/Themes.kt

```diff
@@ -0,0 +1,208 @@
+  val onError = Color(0xffffffff)
+  val onErrorDark = Color(0xff000000)
+  val onPrimary = Color(0xffffffff)
+  val onPrimaryDark = Color(0xff000000)
+  val onSecondary = Color(0xffffffff)
+  val onSecondaryDark = Color(0xff000000)
+  val onSurface = Color(0xff000000)
+  val onSurfaceDark = Color(0xffffffff)
+  val primary = Color(0xffe91e63)
+  val primaryDark = Color(0xfff48fb0)
+  val primaryVariant = Color(0xffc2185b)
+  val primaryVariantDark = Color(0xffc2185b)
+  val secondary = Color(0xffffc107)
+  val secondaryDark = Color(0xffffe082)
+  val secondaryVariant = Color(0xffdda600)
+  val secondaryVariantDark = Color(0xffffe082)
+  val surface = Color(0xffffffff)
+  val surfaceDark = Color(0xff121212)
+
+  val mediumEmphasisGray = Color(0xff808080)
+  val inputGray = Color(0xfff8f9fa)
+  val transparent = Color(0x00000000)
+
+  val lightThemeColors = lightColors(
+      primary = primary,
+      primaryVariant = primaryVariant,
+      secondary = secondary,
+      secondaryVariant = secondaryVariant,
+      background = background,
+      surface = surface,
+      onPrimary = onPrimary,
+      onSecondary = onSecondary,
+      onBackground = onBackgroundLight,
+      onSurface = onSurface,
+      error = error,
+      onError = onError,
+  )
+
+  val darkThemeColors = darkColors(
+      primary = primaryDark,
+      primaryVariant = primaryVariantDark,
+      secondary = secondaryDark,
+      secondaryVariant = secondaryVariantDark,
+      background = backgroundDark,
+      surface = surfaceDark,
+      onPrimary = onPrimaryDark,
+      onSecondary = onSecondaryDark,
+      onBackground = onBackgroundDark,
+      onSurface = onSurfaceDark,
+      error = errorDark,
+      onError = onErrorDark,
+  )
+}
+
+object Typographies {
+  private val robotoFamily = FontFamily(
+      Font(resId = R.font.roboto_regular, weight = FontWeight.Normal, style = FontStyle.Normal),
+      Font(resId = R.font.roboto_bold, weight = FontWeight.Bold, style = FontStyle.Normal),
+      Font(resId = R.font.roboto_light, weight = FontWeight.Light, style = FontStyle.Normal),
+      Font(resId = R.font.roboto_thin, weight = FontWeight.Thin, style = FontStyle.Normal),
+      Font(resId = R.font.roboto_medium, weight = FontWeight.Medium, style = FontStyle.Normal),
+      Font(resId = R.font.roboto_italic, weight = FontWeight.Normal, style = FontStyle.Italic),
+      Font(resId = R.font.roboto_bold_italic, weight = FontWeight.Bold, style = FontStyle.Italic),
+      Font(resId = R.font.roboto_light_italic, weight = FontWeight.Light, style = FontStyle.Italic),
+      Font(resId = R.font.roboto_thin_italic, weight = FontWeight.Thin, style = FontStyle.Italic),
+      Font(resId = R.font.roboto_medium_italic, weight = FontWeight.Medium, style = FontStyle.Italic),
+  )
+
+  val h1TextStyle = TextStyle(
+      fontFamily = robotoFamily,
+      fontWeight = FontWeight.Normal,
+      fontSize = 96.sp,
+      letterSpacing = (-1.5).sp,
```

Comment:
isn't this defined in material?

Reply:
I don't know if our guidelines are the same as material definitions